a11y-meta-display-guide/2.0/draft/guidelines/index.html

```diff
@@ -134,53 +136,51 @@
 			<h2>Introduction</h2>
 
 			<p>Reading a digital publication is a very personal experience. For most people this is routine, and little
-				consideration is given to how the title was obtained before it is read. Users may go to a bookstore,
+				consideration is given to how the title was obtained before it is read. Users may go to a bookstore or library,
 				search for the title to purchase online, or have the title selected for them by an instructor for a
 				class.</p>
 
 			<p>Now consider that the person is blind and relies on assistive technology. The user needs that technology
 				to assist them in the purchase process as well as to read the e-book. The person may wonder: will
 				the screen reader work with this title; are there image descriptions that will be spoken to describe
-				these images; are there page numbers which are accessible; is the reading order correct so a caution
-				after reading a paragraph which could be dangerous will be announced? All of these, and more
-				accessibility concerns are potential issues consumers have when trying to purchase and ultimately read a
+				these images; are there page numbers which are accessible; is the reading order correct so a caution will be announced before reading the paragraph which could be dangerous? All of these accessibility concerns are potential issues consumers have when trying to purchase and ultimately read a
```

Comment:
I understand the intent, but this amusingly sounds like reading paragraphs is dangerous.

Maybe the focus could be returned to the content, so phrase this as "is the reading order correct so a caution is announced before dangerous instructions".